com.netxms.Client.metainfo.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+   <id>com.netxms.Client</id>
```

Comment:
`Client` isn't a great name.

Reply:
NetXMS is a single product, it have different frontends (desktop, web, mobile) which we call clients in the documentation; these clients connects to the backend over the network. Just like IRC Client, or Mail Client. Since only desktop client will be published this way, there will be no ambiguity. I don't have any objections on changing the app-id, but don't see better alternative.

Reply:
then NetXMSClient is the name. (capitalization at your discretion but be aware it can't be changed).

Reply:
done